README.md

```diff
@@ -7,6 +7,8 @@ This is the source repository for the Baïkal CalDAV and CardDAV server.
 
 Head to [sabre.io/baikal][2] for information about installation, upgrading and troubleshooting.
 
+A German Tutorial about installation of Baïkal, using it together with Thunderbird, Android and DavX5, using calendars together/calendar sharing and such as database maintanance and security can be found [here][6].  
```

Comment:
Tutorial -> tutorial

Furthermore I don't get the sentence itself: .... , using calendars together/calendar sharing and such as ...

Thanks a lot!

Reply:
```suggestion
A German Tutorial about installation of Baïkal, using it together with Thunderbird, Android and DavX5, sharing calendars, database maintenance and security can be found [here][6].  
```
Maybe text like this?

Reply:
> using calendars together/calendar sharing 

This refers to https://github.com/sabre-io/Baikal/issues/875#issuecomment-598901532
How would you express this?